src/Event/BasicAuth/BasicAuthInterface.php

```diff
@@ -72,7 +72,7 @@ public function setMatcher($matcher);
     /**
      * Authenticates a request.
      *
-     * @param \Ivory\HttpAdapter\Event\BasicAuth\InternalRequestInterface $internalRequest The internal request.
+     * @param \Ivory\HttpAdapter\Message\InternalRequestInterface $internalRequest The internal request.
```

Comment:
What do you think about omitting the namespace at all here (the class is already known because of the `use` statement)?